admin/class-admin-asset-manager.php

```diff
@@ -503,18 +503,6 @@ protected function styles_to_be_registered() {
 				'src'  => 'yst_plugin_tools-' . $flat_version,
 				'deps' => [ self::PREFIX . 'toggle-switch' ],
```

Comment:
You have removed the 'toggle-switch' asset, but this is still configured as a dependency on this asset.
This dependency needs to be removed.
```suggestion
```